lib/spack/spack/build_systems/cmake.py

```diff
@@ -32,6 +32,50 @@
 from spack.package import PackageBase, run_after
 
 
+def processCMakeCacheFile(cache_file):
```

Comment:
On naming conventions, sure

On the move, I really don't know that anything else is going to be doing this, since I see the scope in which it will be used as only within the function I tend to leave those as nested functions. That's just a preference though